lib/ret_web/channels/auth_channel.ex

```diff
@@ -65,6 +65,12 @@ defmodule RetWeb.AuthChannel do
   def handle_out("auth_credentials" = event, payload, socket) do
     Process.send_after(self(), :close_channel, 1000 * 5)
     push(socket, event, payload)
+
+    # Send the email address over for all connected processes to use as well
+    if socket.assigns |> Map.has_key?(:email) do
+      broadcast!(socket, "auth_email", %{email: socket.assigns.email})
+    end
```

Comment:
Why broadcast this in `handle_out` instead of in `broadcast_credentials_for_identifier_hash`?

Reply:
Oh, right, we're dealing with two separate processes here. nvm.